onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -57,7 +57,7 @@ BasicBackend::BasicBackend(const ONNX_NAMESPACE::ModelProto& model_proto,
         LOGS_DEFAULT(INFO) << log_tag << "Loaded model to the plugin";
       } else {
 #if defined(OPENVINO_2023_0)
-        if (subgraph_context.precision != InferenceEngine::Precision::FP16) {
+        if (!subgraph_context_.has_dynamic_input_shape && dev_prec!="CPU_FP16") {
```

Comment:
Has a bug been raised with OV team for CPU FP 16 Precision

Reply:
Not yet. Shall raise one.